app/src/main/java/com/alphawallet/app/entity/transactionAPI/TransferFetchType.java

```diff
@@ -0,0 +1,11 @@
+package com.alphawallet.app.entity.transactionAPI;
+
+/**
+ * Created by JB on 10/02/2023.
+ */
+public enum TransferFetchType
```

Comment:
Not a big deal but it would be nice to have enums follow a more standard/flexible format

See `ImportWalletType` class

Reply:
Yeah that's good - I didn't know you could do that!

Reply:
Please feel free to refactor when you build on this for OKX.